### PR TITLE
add recipe for forestplot

### PR DESCRIPTION
**Pull request by LSYS**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by LSYS** [comment]
@conda-forge/staged-recipes The checks are failing because the license cannot be found:
`ValueError: License file given in about/license_file (/home/conda/staged-recipes-copy/recipes/forestplot/LICENSE) does not exist in source root dir or in recipe root dir (with meta.yaml)
##[error]Bash exited with code '1'.
`

But I have already included LICENSE in the repo. What should I do?

File: recipes/forestplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - forestplot
+
+about:
+  home: "https://github.com/lsys/forestplot"
+  license: MIT
+  license_family: MIT
+  license_file: https://github.com/LSYS/forestplot/blob/main/LICENSE
```
Comment (ocefpaf) [inline recipes/forestplot/meta.yaml:35]: conda-build won't download that for you. You'll need to include that file here in with your meta.yaml.

File: recipes/forestplot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+-+{% set name = "forestplot" %}
```
Comment (ocefpaf) [inline recipes/forestplot/meta.yaml:1]:
```suggestion
{% set name = "forestplot" %}
```

File: recipes/forestplot/meta.yaml (hunk 1 above)
Comment (LSYS) [reply recipes/forestplot/meta.yaml]: thanks!

File: recipes/forestplot/meta.yaml (hunk 2 above)
Comment (LSYS) [reply recipes/forestplot/meta.yaml]: Thanks!!

**Pull request merged**